terraform/terraform.tfvars

```diff
@@ -0,0 +1,13 @@
+# Mandatory Config Options
+model_name             = "put your model-name here"
+db_application_name    = "put your mongodb app name here"
+certs_application_name = "put your self-signed-certificates app name here"
+nrf_application_name   = "put your nrf app name here"
+
+# Optional Configuration
+channel = "put the charm channel here"
+amf-config = {
+  dnn                   = "put your dnn here"
+  external-amf-ip       = "put your external AMF IP here"
+  external-amf-hostname = "put your external AMF hostname here"
+}
```

Comment:
As they are optional, how can we use the default values? Do we need to remove those lines?

Reply:
I have commented the optional config section in the final patch.

Reply:
Thanks!